index.js

```diff
@@ -17,12 +21,7 @@ function log () {
   }
 }
 
-// TODO: expand default bootstrap list to multiple confirmed & reliable nodes
-if (urlParams.get('bootstrap')) {
-  hyperswarmWebOpts.bootstrap = ["wss://swarm.cblgh.org"].concat(urlParams.get("bootstrap"))
-}
-
-client.addCabal(key, { swarm: hyperswarmWebOpts }).then((details) => {
```

Comment:
These swarm options are not used by `client.addCabal()`.

Reply:
ah ya! good catch, i think i changed it when wrapping up the release of cabal-client from wip status 👍